torchvision/prototype/models/depth/stereo/crestereo.py

```diff
@@ -0,0 +1,1021 @@
+import math
+from functools import partial
+from typing import Iterable, List, Optional, Callable, Tuple, Dict, Union
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision.models.optical_flow.raft as raft
+from torch import Tensor
+from torchvision.models._api import WeightsEnum
+from torchvision.models.optical_flow._utils import make_coords_grid, grid_sample, upsample_flow
+from torchvision.ops import Conv2dNormActivation
+
```

Comment:
please add __all__ = (
    "CREStereo",
   "CREStereo_Weights",
    "crestereo_base",
)
 
in this section.